aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -900,7 +899,17 @@ impl AptosVM {
             &function,
             struct_constructors_enabled,
         )?;
-        session.execute_entry_function(
+
+        if !function.is_entry() {
```

Comment:
Could this be a tiny helper on the function itself? and nit to remove the white space to execute_loaded_function, because now we don't have execute_entry_function and this check is intrinsically related to the execute_loaded_function below (as a precondition).

Reply:
Pre-condition only for entries, yes. Added `is_entry_or_err` helper👍